conman/routes/managers.py

```diff
@@ -30,3 +32,31 @@ def best_match_for_path(self, path):
         except IndexError:
             msg = 'No matching Route for URL. (Have you made a root Route?)'
             raise self.model.DoesNotExist(msg)
+
+    def move_branch(self, old_url, new_url):
+        """
+        Move this Route and all descendants to a new URL.
```

Comment:
If this matches your intention here, I'm surprised this is a `Manager` method and not a `Model` method.